webapp/src/main/java/com/box/l10n/mojito/rest/ai/AIPromptWS.java

```diff
@@ -24,13 +26,19 @@ public class AIPromptWS {
 
   @Autowired PromptService promptService;
 
-  @RequestMapping(value = "/api/ai/prompts", method = RequestMethod.POST)
+  @Operation(summary = "Create a new Prompt")
+  @RequestMapping(
+      value = "/api/ai/prompts",
+      method = RequestMethod.POST,
+      consumes = MediaType.APPLICATION_JSON_VALUE,
+      produces = {MediaType.TEXT_PLAIN_VALUE, MediaType.APPLICATION_JSON_VALUE})
```

Comment:
Is updating the `produces` to include `application/json` the fix for the issue we encountered? If so do we need `text/plain` here?

Have you validated that other endpoints are working as expected?

Reply:
I removed all the consumes and produces